ansible/playbooks/service/discourse.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+- name: Manage Discourse service
+  hosts: [ 'debops_service_discourse' ]
+  become: True
+
+  environment: '{{ inventory__environment | d({})
+                   | combine(inventory__group_environment | d({}))
+                   | combine(inventory__host_environment  | d({})) }}'
+
+  roles:
+
+    - role: debops.discourse/env
+      tags: [ 'role::discourse', 'role::postgresql' ]
+
+    - role: debops.system_groups
+      tags: [ 'role::system_groups', 'skip::system_groups' ]
+
+    - role: debops.apt_preferences
+      tags: [ 'role::apt_preferences', 'skip::apt_preferences' ]
+      apt_preferences__dependent_list:
+        - '{{ nginx__apt_preferences__dependent_list }}'
+        - '{{ nodejs__apt_preferences__dependent_list }}'
+        - '{{ postgresql__apt_preferences__dependent_list }}'
+        - '{{ ruby__apt_preferences__dependent_list }}'
+        - '{{ discourse__apt_preferences__dependent_list }}'
+
+    - role: debops.nodejs
+      tags: [ 'role::nodejs', 'skip::nodejs' ]
+      nodejs__upstream: True
+      nodejs__yarn_upstream: True
```

Comment:
With the recent changes in `debops.nodejs`, variables that enable upstream support shouldn't be needed, at least on Debian Buster. Can you check and confirm?